aea/cli/add.py

```diff
@@ -212,7 +213,13 @@ def fetch_item_remote(
         except ImportError:
             click.echo("Please install IPFS plugin.")
 
-        return fetch_ipfs(item_type, public_id=item_public_id, cwd=cwd, dest=dest)
+        package_path = fetch_ipfs(
+            item_type, public_id=item_public_id, cwd=cwd, dest=dest
+        )
+        if package_path is not None:
+            return package_path
+        click.echo(f"Cannot find hash for {item_public_id}")
+        click.echo("Fetching item using http.")
```

Comment:
Right now I've used HTTP as a fail-safe for ipfs instead of fetching locally or locally distributed so it would keep the whole remote context intact. If the HTTP will also fail the user can specify `--local` flag.